handbook/engineering/search/iteration_log.md

```diff
@@ -14,9 +14,41 @@ This document contains the goals and work log for the search team's [2-week iter
 
 ## 2020-09-21 to 2020-10-02
 
-### $GOAL_OR_THEME
+### Scale indexed search to 500k repositories
 
-- **Owner(s):** $OWNER
+- **Owner(s):** Stefan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
+- **Work log:**
+    - YYYY-MM-DD: $UPDATE
+
+### Streaming search
+
+- **Owner(s):** Keegan
+- **Outcomes:**
+    - $DESIRED_OUTCOME
```

Comment:
@keegancsmith can you describe what you plan to accomplish on streaming search for the next iteration? (at this point we need to get a frontend engineer involved to work with you on this, right?)